File: Go4Lunch/app/src/main/java/com/openclassrooms/go4lunch/data/model/api/places/nearbysearch/Photo.java
package com.openclassrooms.go4lunch.data.model.api.places.nearbysearch;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

/**
 * Photo Class from Google Place Nearby Search Api
 */
public class Photo {

    @SerializedName("height")
    @Expose
    private int height;

    @SerializedName("html_attributions")
    @Expose
    private List<String> htmlAttributions = null;

    @SerializedName("photo_reference")
    @Expose
    private String photoReference;

    @SerializedName("width")
    @Expose
    private int width;

    public int getHeight() {
        return height;
    }

    public void setHeight(int height) {
        this.height = height;
    }

    public String getPhotoReference() {
        return photoReference;
    }

    public int getWidth() {
        return width;
    }

    public void setWidth(int width) {
        this.width = width;
    }

}

File: RealEstateManager/app/src/main/java/com/openclassrooms/realestatemanager/models/place/detail/Photo.kt
package com.openclassrooms.realestatemanager.models.place.detail

import com.google.gson.annotations.SerializedName

data class Photo (
    val height: Long,

    @SerializedName(value = "html_attributions")
    val htmlAttributions: List<String>,

    @SerializedName(value = "photo_reference")
    val photoReference: String,

    val width: Long
)